doc/theming.rst

```diff
@@ -116,6 +116,12 @@ These themes are:
 
   - **sidebarwidth** (an integer): Width of the sidebar in pixels.  (Do not
     include ``px`` in the value.)  Defaults to 230 pixels.
+  
+  - **canonical_url** (sting): A `canonical url`_ to let search engines know
+    they should give higher ranking to latest version of the docs.  The url
+    points to the root of the documentation and requires a trailing slash.
+
+.. cronical url: https://en.wikipedia.org/wiki/Canonical_link_element
```

Comment:
Did you build this document ever? A hyperlink target should start with underscore. And this also has a typo. Please check them please.

BTW, I prefer to use [RFC 6596](https://tools.ietf.org/html/rfc6596) instead. What do you think?

Reply:
I do not have a strong preference either way, but I think using the RFC 6596 link could be too technical for some.